specutils/spectra/spectrum1d.py

```diff
@@ -246,9 +246,26 @@ def __getitem__(self, item):
         The first case is handled by the parent class, while the second is
         handled here.
         """
+
         if len(self.flux.shape) > 1:
```

Comment:
```suggestion
        if self.flux.ndim > 1 or (type(item) == tuple and item[0] == Ellipsis):
```
This would enable the `[..., lower, upper]` syntax for 1D and multi-D flux spectra alike.